.gitignore

```diff
@@ -12,4 +12,7 @@ __pycache__/
 # Coverage.py
 .coverage
 coverage/
+build/**
+dist/**
+release-dist/**
```

Comment:
```suggestion
release-dist/
```

Reply:
FWIW I'd move these further away from `# Coverage.py` as they don't belong to the same group.